packages/config/config/devices/0x0098/ct101.json

```diff
@@ -148,14 +243,65 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Fast recovery Mode",
+					"label": "Fast recovery mode",
 					"value": 1
 				},
 				{
-					"label": "Economy recovery Mode",
+					"label": "Economy recovery mode",
 					"value": 2
 				}
 			]
+		},
+		"10": {
+			"label": "Temperature Reporting Filter",
+			"description": "Upper/Lower bounds for thermostat temperature reporting",
```

Comment:
This description doesn't appear to make sense. How can this be both the upper and lower bounds? Please referecnce the manual.

Reply:
Looks like this may need to be broken down into partial parameters. Reading up on that now.

Reply:
This param is complicated with different layouts between get & set. Since there doesn't appear to be a known real-world use for it, I've removed it from the config.